scalableminds/webknossos
# Fix import of N5 datasets by adding n5 schema to frontend validation

### URL of deployed dev instance (used for testing):
- https://fixn5schema.webknossos.xyz

### Steps to test:
- try to import `s3://janelia-cosem-datasets/jrc_hela-3/jrc_hela-3.n5/em/fibsem-uint16`
- you should be able to see data after import (might be a bit slow)

------
(Please delete unneeded items, merge only when none are left open)
